loadtest/control/simplecontroller/controller.go

```diff
@@ -33,6 +35,8 @@ func New(id int, user user.User, status chan<- control.UserStatus) *SimpleContro
 	}
 }
 
+// Run begins performing a set of actions in a loop with a defined wait
+// in between the actions. It keeps on doing it until Stop is invoked.
```

Comment:
I would only add that this is a blocking function and it's meant to be used inside a goroutine.